lti_consumer/lti_consumer.py

```diff
@@ -822,17 +819,17 @@ def _get_context_for_template(self):
             'element_id': self.location.html_id(),  # pylint: disable=no-member
             'element_class': self.category,  # pylint: disable=no-member
             'launch_target': self.launch_target,
-            'display_name': self.display_name,
+            'display_name': self.ugettext(self.display_name),
             'form_url': self.runtime.handler_url(self, 'lti_launch_handler').rstrip('/?'),
             'hide_launch': self.hide_launch,
             'has_score': self.has_score,
             'weight': self.weight,
             'module_score': self.module_score,
-            'comment': sanitized_comment,
+            'comment': self.ugettext(sanitized_comment),
             'description': self.description,
             'ask_to_send_username': self.ask_to_send_username,
             'ask_to_send_email': self.ask_to_send_email,
-            'button_text': self.button_text,
+            'button_text': self.ugettext(self.button_text),
```

Comment:
@douglashall you are right but above, The purpose was generating a .po file. Now here runtime service is trying to translate the dynamic text if its translations are available in XBlock message catalogs.
@mattdrayer ^^